frontend/package.json

```diff
@@ -102,7 +102,7 @@
     "react-mailchimp-subscribe": "^2.1.3",
     "react-markdown": "^8.0.3",
     "react-redux": "^8.0.2",
-    "react-select": "^5.8.1",
+    "react-select": "^5.8.3",
```

Comment:
Question: Why was patch version upgrade necessary?

Reply:
you're right, it wasnt needed 🤔